### PR TITLE
Adds diff_drive plugin. Launches node to move the robot.

### PR DESCRIPTION
**Pull request by francocipollone**
- model's inertia is updated.
- `diff_drive_plugin` is added to the model.
- `rqt_steering` node is launched with the world.
- `teleop_twist_keyboard` node is launched with the world.

### CODE REVIEW

File: gazebo_worlds/launch/gazebo.launch [hunk 1]
```diff
@@ -8,4 +8,8 @@
 
   <node name="robot_spawn" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen"
   args="-file $(find cipobot)/model/cipobot/model.sdf -sdf -x 0.0 -y 0.0 -z 0.0 -R 0 -P 0 -Y 0 -model cipobot" />
+
+  <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py"/>
+  <node name="rqt_robot_steering" pkg="rqt_robot_steering" type="rqt_robot_steering"/>
```
Comment (BarceloChristian) [inline gazebo_worlds/launch/gazebo.launch:13]: rqt_robot_steering has `/r2d2/cmd_vel` as default topic, could you add a remap here fort that topic to go through `/cmd_vel`?

Comment (francocipollone) [reply gazebo_worlds/launch/gazebo.launch:13]: Ah, all right. Is it odd that beyond the topic is wrong it works anyway?

Comment (francocipollone) [reply gazebo_worlds/launch/gazebo.launch:13]: I have checked the `rqt_robot_steering` node and it is publishing in `/cmd_vel`. What am I missing?

Screenshot from 2020-05-12 20-28-52

Comment (BarceloChristian) [reply gazebo_worlds/launch/gazebo.launch:13]: It might be a miss-configuration in my set-up. Sorry for the misunderstanding, and the late reply.

**Pull request merged**